Servers/Atlas/.arc/submit.py

```diff
@@ -62,12 +62,18 @@
         # will be renamed to ``job.sh`` when uploaded
         'gaussian_job': """#!/bin/csh
 
-touch start
+touch initial_time
+
+mkdir -p $GAUSS_SCRDIR
 
 source /Local/ce_dana/g09/bsd/g09.login
 
 /Local/ce_dana/g09/g09 < input.gjf > input.log
```

Comment:
I think it may be good now to change input.log to output.out straight away instead of getting ARC to deal with it? I know some people get confused when they're looking in their gaussian folder and they see input.log but not output.out. I think it can also help arc restart